src/lib/components/DashNgl.react.js

```diff
@@ -225,25 +184,32 @@ DashNgl.defaultProps = {
 
 DashNgl.propTypes = {
     /**
-     * The ID used to identify this component in Dash callbacks.
+     * The ID of this component, used to identify dash components in
+     * callbacks. The ID needs to be unique across all of the
+     * components in an app.
      */
     id: PropTypes.string,
 
     /**
-     * CSS styling for viewport container
+     * The height (in px) and the width (in %) of the container
```

Comment:
This is is on purpose because I did not manage to get an automated resizing of the width of the ngl viewer using pixels. But this whole section has to be improved any way...
I am actually not using that variable but hard coding the values in the render function because somehow the render function does not see `const defaultViewportStyle`

In my component it works https://github.com/IvoLeist/dash_ngl
Do you have any what is the reason for that ?

Reply:
You could try using the Chrome debugger to read the values of `height` and `width` whenever the `render` function runs (or you could also just print it). My guess is that you're setting the height and width somewhere in your demo app, so when the initial callback gets fired, it sends a `None` value which might override the defaults.

Reply:
The problem is not that it is overwritten but rather it does not compile correctly any longer or especially does not pass your JavaScript lint test. Somehow the render function does literally not see the variable. I guess I need to somehow "import" the variable into the render function but so far I have not figured how...

So weird that it in my component it just works (?)
But to be honest I did not do any JavaScript linting and such a like.